react/ProductSearchContextProvider.js

```diff
@@ -35,6 +35,14 @@ class ProductSearchContextProvider extends Component {
       order: PropTypes.oneOf(SORT_OPTIONS.map(o => o.value)),
       priceRange: PropTypes.string,
     }),
+    /** Custom query `query` param */
+    queryField: PropTypes.string,
```

Comment:
and the query object above? It will be deprecated?

Reply:
no, these `*Field` props are used for the custom search query, which I don't think makes sense anymore because we have the exact same functionality in the search result component.